packages/emotion-primitives/README.md

```diff
@@ -0,0 +1,90 @@
+# Emotion primitives
+
+> Style and render primitive interfaces across multiple targets with emotion
+
+## Introduction
+
+Emotion primitives makes it easy to style and render primitives across multiple platforms like `Web`, `React Native` and `Sketch` using the similar `emotion` API.
+
+## Install
+
+```
+npm install emotion-primitives
+```
+
+or if you use yarn
+
+```
+yarn add emotion-primitives
+```
+
+This package also depends on `react` and `react-primitives` so make sure you've them installed.
+
+## Example
+
+```js
+import React from 'react'
+import emotion from 'emotion-primitives'
```

Comment:
Could you change this to be imported as styled so that syntax highlighting and etc. just work?

Reply:
Sure!